CHANGELOG.md

```diff
@@ -1,3 +1,704 @@
+## [theila 1.0.0](https://github.com/siderolabs/theila/releases/tag/v1.0.0) (2022-03-29)
+
+Welcome to the v1.0.0 release of theila!
+
+
+
+Please try out the release binaries and report any issues at
+https://github.com/siderolabs/theila/issues.
+
+### The New UI
+
+Sidero UI is getting the next iteration of the design.
+The color scheme was aligned to the new Sidero company color scheme.
+
+Main changes UX-wise:
+- all clusters are now placed in the context dropdown: both kubeconfig clusters and CAPI clusters.
+- now it is possible to see Kubernetes upgrades history (dropped after theila restart as of now).
+- pod information has got more details.
+- servers list is now also available for CAPI clusters.
+- now the UI provides summary page for cluster resources usage.
+
+
+### Contributors
```

Comment:
is the contributor list here correct, seems like the one for talos?

Reply:
I think it gathers contributors from all deps, and Talos is one of them, so I guess it's correct.